src/main/minio.js

```diff
@@ -410,7 +410,13 @@ export class Client {
         }
         this.logHTTP(reqOptions, response)
         if (returnResponse) return cb(null, response)
-        response.socket.end()
+        if (response.socket) {
+          // For node environment
+          response.socket.end()
+        } else {
+          // For browserify environment
+          response.on('data', ()=>{})
```

Comment:
it will work, but `socket.end()` would be the right way to close the socket. `response.on('data', ()=>{})` is a workaround on browser. So it is better to do it this way, I think. (Though I see the advantage of having `response.on('data', ()=>{})` for both node and browser is that you don't need if/else statements.)

Reply:
yes that is what i was thinking. Accessing socket feels too low level IMO.

Reply:
> yes that is what i was thinking. Accessing socket feels too low level IMO. You let nodejs runtime to do that for you but our approach is just to drain the response body.

do you mean nodejs runtime will *automatically* close underlying socket when it frees response ?

Reply:
Well that is what I would expect. That would allow underlying http to reuse the socket.